src/coreclr/nativeaot/Runtime/thread.h

```diff
@@ -88,8 +88,7 @@ struct ThreadBuffer
     uintptr_t               m_uHijackedReturnValueFlags;            
     PTR_ExInfo              m_pExInfoStackHead;
     Object*                 m_threadAbortException;                 // ThreadAbortException instance -set only during thread abort
-    PTR_PTR_VOID            m_pThreadLocalModuleStatics;
-    uint32_t                m_numThreadLocalModuleStatics;
+    Object*                 m_pThreadLocalModuleStatics;
```

Comment:
Do we need to report this to the GC now that it is a raw `Object*` ?

Reply:
Yes it is reported as one of the roots that belong to the thread.